BASE/.vsts/linux-build.yml

```diff
@@ -1,47 +1,50 @@
 pool:
   vmImage: 'ubuntu-16.04'
```

Comment:
lets move to newer version. 16 is getting older.. 18.04.5

Reply:
https://wiki.ubuntu.com/Releases

Reply:
i tried...using "ubuntu-latest" but two tests started to fail in MetricsTest

Reply:
:D lets pick 18.04.5 and try. If it fails, then we have an issue with latest linux.

Reply:
today, ubuntu-latest = 18.04